src/util/redux.js

```diff
@@ -0,0 +1,246 @@
+import {
```

Comment:
i'm fine with it as long we're consistent and change it it all the source files, so maybe save this for another PR